### PR TITLE
Spotify Docs

### PR DESCRIPTION
**Pull request by erin2722**
🎧  Here are docs for my new Spotify Node!

### CODE REVIEW

File: docs/nodes/credentials/Spotify/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Authentication
+
+## Pre-requisites
+
+Create a [Spotify Developer](https://developer.spotify.com/dashboard/login) account.
+
+## Steps
+
+1. Open your Spotify for Developers dashboard.
+2. Create an app.
```
Comment (tanay1337) [inline docs/nodes/credentials/Spotify/README.md:13]: Could you please make this more specific? Is there a button you have to click on to do this?

File: docs/nodes/credentials/Spotify/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Spotify
```
Comment (tanay1337) [inline docs/nodes/credentials/Spotify/README.md:1]: Could you please make sure that the formatting of the document aligns with the updated template?

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
+---
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Spotify/README.md:1]: Could you please make sure that the formatting of the document aligns with the updated template?

File: docs/nodes/credentials/Spotify/README.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+## Steps
+
+1. Open your Spotify for Developers dashboard.
+2. Create an app.
+3. On the app's dashboard, copy the Client ID and the Client Secret.
+4. Open the app settings and add the Callback URL provided by n8n to the Redirect URIs list.
+5. Use the Client ID and the Client Secret to authenticate with n8n.
+
+
+![The Spotify App Dashboard](https://i.imgur.com/qmjR3xq.gif)
```
Comment (tanay1337) [inline docs/nodes/credentials/Spotify/README.md:19]: Could you please add an empty newline at the end of all the markdown files?

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+### 2. Spotify node
+
+1. First of all, you'll have to enter credentials for the Spotify node. You can find out how to do that [here](../../../credentials/Spotify/README.md).
+2. Select the *Player* Resource.
+3. Select the *Add Song to Queue* Operation.
+4. Enter in the song's URI (or ID) to the *Track ID* field. The GIF demonstrates how to find the Track ID on Spotify's UI.
+5. Click on *Execute Node* to run the workflow.
+
+![Spotify URI](https://i.imgur.com/e0Q3PQ7.gif)
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Spotify/README.md:62]: Could you please add an empty newline at the end of all the markdown files?

Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Spotify/README.md:62]: Could you please make sure that this image exists locally and loads from there (as opposed to from imgur)?

File: docs/nodes/credentials/Spotify/README.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+# Authentication
+
+## Pre-requisites
+
+Create a [Spotify Developer](https://developer.spotify.com/dashboard/login) account.
+
+## Steps
+
+1. Open your Spotify for Developers dashboard.
```
Comment (tanay1337) [inline docs/nodes/credentials/Spotify/README.md:12]: Could you please add a link for that here?

File: docs/nodes/credentials/Spotify/README.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+2. Create an app.
+3. On the app's dashboard, copy the Client ID and the Client Secret.
+4. Open the app settings and add the Callback URL provided by n8n to the Redirect URIs list.
```
Comment (tanay1337) [inline docs/nodes/credentials/Spotify/README.md:15]: Could you please check if it is called 'Callback URL' in the Editor UI?

File: docs/nodes/credentials/Spotify/README.md (hunk 4 above)
Comment (tanay1337) [inline docs/nodes/credentials/Spotify/README.md:19]: Could you please make sure that this image exists locally and loads from there (as opposed to from imgur)?

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. Spotify node
+
+1. First of all, you'll have to enter credentials for the Spotify node. You can find out how to do that [here](../../../credentials/Spotify/README.md).
+2. Select the *Player* Resource.
+3. Select the *Add Song to Queue* Operation.
+4. Enter in the song's URI (or ID) to the *Track ID* field. The GIF demonstrates how to find the Track ID on Spotify's UI.
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Spotify/README.md:59]: 'The following GIF shows you how to ....'

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+1. First of all, you'll have to enter credentials for the Spotify node. You can find out how to do that [here](../../../credentials/Spotify/README.md).
+2. Select the *Player* Resource.
+3. Select the *Add Song to Queue* Operation.
+4. Enter the song's URI (or ID) to the *Track ID* field. The following GIF shows you how to find the Track ID on Spotify's UI.
+5. Click on *Execute Node* to run the workflow.
+
+![Spotify URI](./spotifyURI.gif)
+
+
+## Further Reading
```
Comment (tanay1337) [inline docs/nodes/nodes-library/nodes/Spotify/README.md:66]: This probably got added by mistake?

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+    - Start your music
+- Playlist
+    - Add an item to a playlist
+    - Get a playlist
+    - Get a playlist's tracks
+    - Get a user's playlists
+    - Remove an item from a playlist
+- Track
+    - Get a track
+    - Get a track's audio features
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/Spotify/README.md:39]: Could you make sure that this matches the descriptions in the node?
image

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. Spotify node
+
+1. First of all, you'll have to enter credentials for the Spotify node. You can find out how to do that [here](../../../credentials/Spotify/README.md).
+2. Select the *Player* Resource.
+3. Select the *Add Song to Queue* Operation.
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/Spotify/README.md:59]: Could you please remove these steps, just checked and they appear to be the default options for that node.

File: docs/nodes/nodes-library/nodes/Spotify/README.md [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+4. Enter the song's URI (or ID) to the *Track ID* field. The following GIF shows you how to find the Track ID on Spotify's UI.
+5. Click on *Execute Node* to run the workflow.
+
+![Spotify URI](./spotifyURI.gif)
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/Spotify/README.md:63]: Please end all your files with a newline

File: docs/nodes/credentials/Spotify/README.md [hunk 13]
```diff
@@ -0,0 +1,17 @@
...
+## Pre-requisites
+
+Create a [Spotify Developer](https://developer.spotify.com/dashboard/login) account.
+
+## Using OAuth
+
+1. Access your [Spotify for Developers](https://developer.spotify.com/dashboard/login) dashboard.
+2. Click the *Create an App* button at the top of your dashboard. Enter in the app's name and description and click *Create*.
+3. Use provided Client Secret and Client ID with your Spotify node credentials in n8n.
+4. Open the app settings and add the n8n provided redirect URL to the app's redirect URI list. [Redirect URL Explanation here](../README.md).
```
Comment (smamudhan) [inline docs/nodes/credentials/Spotify/README.md:15]: Could you just link the word "here" instead of the entire statement?

**Comment by erin2722** [comment]
Hi @smamudhan! Thanks for the feedback-- done!

**Pull request merged**